webcompat/webhooks/model.py

```diff
@@ -181,3 +198,97 @@ def get_public_issue_number(self):
         if url:
             url = self.public_url.strip().rsplit('/', 1)[1]
         return url
+
+    def process_issue_action(self):
```

Comment:
I so want to change this process_issue_action, but I don't know yet how exactly 🤣